src/lib/mongoBackend/MongoCommonRegister.cpp

```diff
@@ -82,7 +83,7 @@ static bool processSubscriptions
                                          trigs->attrL,
                                          mapSubId,
                                          trigs->reference,
-                                         trigs->format,
+                                         JSON,
```

Comment:
In a few places, the parameter *format* has disappeared and *notifyFormat* appeared in its place, with a different type.
I didn't want to touch ngsi9 stuff during the PR (it would never end!) so I believe hardcoding JSON is the best choice (which will be fixed once ngsi9 is handled the same way).